dev/serializer_context.h

```diff
@@ -19,20 +19,6 @@ namespace sqlite_orm {
 
             serializer_context(const db_objects_type& dbObjects) : db_objects{dbObjects} {}
         };
-
-        template<class S>
-        struct serializer_context_builder {
```

Comment:
it is very good that we've removed this extra class. Less abstractions - easier code